### PR TITLE
Coverity 02

### PR DESCRIPTION
**Pull request by andrewbird**
Second chunk of Coverity fixes. Could you please review carefully the switch/missing break fixes as they are open to interpretation?

### CODE REVIEW

File: src/base/core/int.c [hunk 1]
```diff
@@ -864,6 +864,7 @@ static int int15(void)
     case 0x81:			/* default BIOS hook: device close */
     case 0x82:			/* default BIOS hook: program termination */
 	HI(ax) = 0;
+	/* Fall through */
```
Comment (stsp) [inline src/base/core/int.c:867]: I think break is actually missing here.
Setting AH to 0 and CARRY at the
same time, probably makes no sense.

Comment (skitt) [reply src/base/core/int.c:867]: Agreed, see https://fd.lod.bz/rbil/interrup/bios/1582.html

Comment (andrewbird) [reply src/base/core/int.c:867]: Thanks, I'll fix that up!

**Comment by stsp** [comment]
Thanks!

**Pull request merged**